app/controllers/api/v1/merchants/customers_controller.rb

```diff
@@ -0,0 +1,9 @@
+class Api::V1::Merchants::CustomersController < ApplicationController
```

Comment:
@icorson3 @jmejia is it ok that we put pending customer invoices and favorite customer in the same controller like this?